### PR TITLE
Population exposed feature enhancements

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This PR isolates Exposed population analysis into a different result with different parameters. This allows considering exposed population not dependent of a provided basline layer, performing multiple feature geometries on a single admin boundary. 

**How to test:**
- modify file src/context/analysisResultStateSlice.ts ANALYSIS_API_URL variable to point to your local API instance.

**Missing:**
- DataTable generation from api results.

image

### CODE REVIEW

**Review by wadhwamatic — COMMENTED** [review]
Thanks for the instructions on testing. Looks good from a functionality perspective including the structure of the response to eventually provide a rendered table. Let's wait for code review from @ericboucher.

File: src/context/analysisResultStateSlice.ts [hunk 1]
```diff
@@ -138,14 +143,109 @@ function generateTableFromApiData(
...
+  )) as Feature[];
+
+  const stats: number[] = Array.prototype.sort.call(
+    features.map(f =>
+      f.properties && f.properties[statistic] ? f.properties[statistic] : 0,
+    ),
+    (a, b) => a - b, // This function is required since JS assumes unicode values.
+  );
+
+  const statsLen = stats.length;
```
Comment (ericboucher) [inline src/context/analysisResultStateSlice.ts:231]: Please explain what this does

**Review by ericboucher — COMMENTED** [review]
Looks good overall, please add comments in the code when we are doing non-obvious operations. Especially in the API, thanks :)

**Comment by JorgeMartinezG** [comment]
@ericboucher Included comments in commits 445a123 and 1b698db

**Review by wadhwamatic — APPROVED** [review]
Thanks for adding comments as Eric suggested. Tested and approved.

**Pull request merged**